fn_utilities/fn_utilities/util/customize.py

```diff
@@ -51,7 +51,6 @@ def customization_data(client=None):
     #     resilient_search_template
     #     resilient_url
     #     rest_body
-    #     rest_cookies
```

Comment:
I don't think this should be removed? Was this input deleted by mistake?

Reply:
Yes, removed by mistake, not really sure though how has it got deleted.

Reply:
If the function that uses this input was modified in the UI, it may have been deleted.

Does regenerating a customize.py file fix the issue?

Reply:
Ideally it does, and just FYI that it is regenerated only. Next time I'll check again.

Reply:
after regenerate rest_cookies is still missing